config/initializers/bullet.rb

```diff
@@ -0,0 +1,10 @@
+unless Rails.env.production?
+  Rails.application.configure do
+    config.after_initialize do
+      Bullet.enable = true
```

Comment:
Does setting true to all these mean the same logs will be available in all three places?

If that's the case, should we just choose one place to set to true?

Reply:
Yep, that's it. We need to have the config `Bullet.enable` as true and choose another config to display the logs. 

https://github.com/flyerhzm/bullet#configuration

When we were testing, we used more the config `Bullet.bullet_logger`. Maybe, we can just leave the configs `Bullet.enable` and `Bullet.bullet_logger` and, if desirable during the process of development, who want, can activate other configs.

What do you think?

Reply:
Yeah, let's start off with both enable and bullet_logger then!

Reply:
Done! 😄